src/main/Config.cpp

```diff
@@ -31,7 +31,7 @@
 
 namespace stellar
 {
-const uint32 Config::CURRENT_LEDGER_PROTOCOL_VERSION = 18
+const uint32 Config::CURRENT_LEDGER_PROTOCOL_VERSION = 19
```

Comment:
sorry I was wrong on this one: we need to first merge this PR and in a separate one switch to protocol 19 as "current".

reason is that we want to validate that meta checks don't change for protocol 18.

Reply:
Ah yeah. Removed